src/app/payment/paymentMethod/BoltCustomForm.tsx

```diff
@@ -1,16 +1,45 @@
-import React, { FunctionComponent } from 'react';
+import React, { useMemo, FunctionComponent } from 'react';
+
+import { TranslatedHtml, TranslatedString } from '../../locale';
+import { CheckboxFormField } from '../../ui/form';
 
 export interface BoltCustomFormProps {
     containerId: string;
+    showCreateAccountCheckbox: boolean;
 }
 
-const BoltCustomForm: FunctionComponent<BoltCustomFormProps> = ({ containerId }) => {
+const BoltCreateAccountCheckbox: FunctionComponent = () => {
+    const options = {
```

Comment:
Can we create this options variable outside the component, so they don't need to be recreated on every render?

Reply:
Yes we can

Reply:
Updated